core/src/main/java/net/sourceforge/jnlp/cache/ResourceDownloader.java

```diff
@@ -295,8 +296,8 @@ protected UrlRequestResult findBestUrl(final Resource resource) {
                     Map<String, String> requestProperties = new HashMap<>();
                     requestProperties.put("Accept-Encoding", "pack200-gzip, gzip");
```

Comment:
we can directly create constants for the strings

Reply:
This is only step 1 of a larger refactoring.
Extractions of constants I will do later